src/pybind/mgr/dashboard/CMakeLists.txt

```
@@ -49,7 +49,7 @@ endif()
 
 add_npm_command(
   OUTPUT "${CMAKE_SOURCE_DIR}/src/pybind/mgr/dashboard/frontend/node_modules"
-  COMMAND CYPRESS_CACHE_FOLDER=${CMAKE_SOURCE_DIR}/build/src/pybind/mgr/dashboard/cypress npm ci
+  COMMAND CYPRESS_CACHE_FOLDER=${CMAKE_SOURCE_DIR}/build/src/pybind/mgr/dashboard/cypress NG_CLI_ANALYTICS=off npm ci
```

I had removed this previously because angular in recent versions was supposed to be able to read this config from `angular.json`.
Since this isn't always the case I think this is the best solution.

Looking at our previous [code](https://github.com/ceph/ceph/commit/0034971da9b0d1c6a0f610d66cffc6eee1842148#diff-e39dffd2b7f8e7e205f4075fff60743e) and at https://github.com/angular/angular-cli/issues/16784#issuecomment-656610090 it seems  `false` is a better/saffer value for `NG_CLI_ANALYTICS`
```suggestion
  COMMAND CYPRESS_CACHE_FOLDER=${CMAKE_SOURCE_DIR}/build/src/pybind/mgr/dashboard/cypress NG_CLI_ANALYTICS=false npm ci
```

---

Done

---

@badone does this suggestion make sense to you? Would you mind amending the PR accordingly? Thanks!

---

Thought I did? The change was from "off" to "false" right?